src/main/java/by/exadel/internship/team_four/entity/Internship.java

```diff
@@ -0,0 +1,54 @@
+package by.exadel.internship.team_four.entity;
+
+import by.exadel.internship.team_four.entity.enums.FormatOfInternship;
+import by.exadel.internship.team_four.entity.enums.Technology;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDate;
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class Internship {
+
+    private UUID id;
+    private String name;
+    private LocalDate startDateInternship;
+    private LocalDate endDateInternship;
```

Comment:
Это поле уже является частью класса Internship.
```suggestion
    private LocalDate endDate;
```